Moesif.Middleware/Helpers/Tasks.cs

```diff
@@ -58,21 +58,6 @@ static List<EventModel> QueueGetAll(ConcurrentQueue<EventModel> moesifQueue, int
                         var createBatchEventResponse = await client.Api.CreateEventsBatchAsync(batchEvents);
                         createBatchEventResponse = createBatchEventResponse.ToDictionary(k => k.Key.ToLower(), k => k.Value);
 
-                        // Signal events
-                        var configETag = createBatchEventResponse["x-moesif-config-etag"];
```

Comment:
@qmjava , why did we get rid of checking etag? we still want to sync if etag changes? We should also ensure thread safe (dont launch a large number of syncs in parallel)

Reply:
Because there is a dedicated thread for sync both appConfig and governance rule.

Reply:
The bg thread is a failsafe. We still want to push changes as soon as we see an etag change. Thats the whole point of etag. The question earlier is why did etag not work. We even tested sending one event to ensure SDK saw updated etag on response, but still did not reflect.

Reply:
Well, the only issue with this approach is that there are have to be some events. If there are no events, 
they won't get synced.

Reply:
yes, that is why we have both. At scale, we expect etag to be dominant. However, if server is idle, we expect the timed based trigger to be dominant.

Reply:
eTag syncs configs/rules immediately for high frequency ingestion whereas timer-based scheduler works fine for low-frequency.

Reply:
Well, what if there are no events?

Reply:
in that case, nothing to worry about. Scheduler will update the config/rules on its own time.